From 82f725bac371723a1206354ff249d3d6a81bcece Mon Sep 17 00:00:00 2001
From: Alberto Gonzalez <boqwxp@airmail.cc>
Date: Fri, 8 May 2020 20:35:40 +0000
Subject: [PATCH] Use jemalloc.

---
 src/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Makefile b/src/Makefile
index 57b128bd..66ef75e4 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -779,6 +779,7 @@ endif
 ifeq ($(YICES_MODE),release)
 CFLAGS := $(CFLAGS) -O3 -fomit-frame-pointer $(NO_STACK_PROTECTOR)
 CPPFLAGS := $(CPPFLAGS) -DNDEBUG
+LIBS += -ljemalloc
 else
 ifeq ($(YICES_MODE),devel)
 CFLAGS := $(CFLAGS) -O3 -g -fomit-frame-pointer $(NO_STACK_PROTECTOR)
-- 
2.20.1

